README.md

```diff
@@ -4,17 +4,15 @@ UI for citizen-profile - This project was bootstrapped with [Create React App](h
 
 ## Environments
 
-Test: https://profiili.test.kuva.hel.ninja/
+Dev: https://profiili.dev.hel.ninja/
 
-Production: https://profiili.prod.kuva.hel.ninja/
+Test: https://profiili.test.hel.ninja/
 
-## Issues board
-
-https://helsinkisolutionoffice.atlassian.net/projects/OM/issues/?filter=allissues&=
+Production: https://profiili.hel.fi
 
-## CI
+## Issues board
 
-The test-environment is built automatically from `develop`-branch.
+https://helsinkisolutionoffice.atlassian.net/browse/HP-1305?jql=project%20%3D%20%22HP%22%20AND%20component%20%3D%20%22Profiili%20UI%22
```

Comment:
IMO it doesn't make sense to include a URL to an issue database that can't be publicly accessed in an open source project.